pedestal.vase.iml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@mtnygard should we exclude this file from version control?

Reply:
Yes, we should. Unfortunately, git submodules on another project kind of screwed me up... which also introduced the spurious commit 08da4eb

Reply:
I just pushed a new commit that removes the project file.